app/services/gatherers/vote_account_details_service.rb

```diff
@@ -10,14 +10,13 @@ def initialize(network:, config_urls:, always_update: false)
       @always_update = always_update
       @network = network
       @config_urls = config_urls
-      @range_start = VoteAccount.where(network: @network).order(id: :asc).first.id
-      @range_end = VoteAccount.where(network: @network).order(id: :asc).last.id
+      @range_start = VoteAccount.where(network: @network).first.id
```

Comment:
Is it possible that VoteAccounts are not ordered as we expect here?
For example (ids) :  1,2,3,5,6,4
Then @range_start will be 1 and @range_end will be 4. Some VoteAccounts will be skipped in that case.

To be honest I don't see a reason for specifying `where(id: @range_start..@range_end`  down below. I think we can delete that for where clause

Reply:
I agree, this range logic is useless here. I did remove it.